doc/tutorials/dnn/dnn_yolo/dnn_yolo.markdown

```diff
@@ -51,4 +51,151 @@ $ example_dnn_object_detection --config=[PATH-TO-DARKNET]/cfg/yolo.cfg --model=[
 
 @endcode
 
+
+Running pre-trained YOLO model in OpenCV
+----------------------------------------
+
+Deploying pre-trained models is a common task in machine learning, particularly when working with hardware that does not support certain frameworks like PyTorch. This guide provides a comprehensive overview of exporting pre-trained YOLO family models from PyTorch and deploying them using OpenCV's runtime framework. For demonstration purposes, we will focus on the [YOLOX](https://github.com/Megvii-BaseDetection/YOLOX/blob/main/LICENSE) model, but the methodology applies to other supported<sup>*</sup> models.
```

Comment:
it makes sense to point to repo root, but not only license file.

Reply:
Sorry, fixed.